scripts/apollo_format.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright 2020 The Apollo Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+# Usage:
+#   apollo_format.sh [Options] <path/to/src/dir/or/files>
+
+# Fail on error
+set -e
+
+TOP_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+source "${TOP_DIR}/scripts/apollo.bashrc"
+
+function print_usage() {
+    info "Usage: $0 [Options] <path/to/src/dir/or/files> "
+    info "Options:"
+    info "${TAB}-p|--python      Format Python code"
+    info "${TAB}-b|--bazel       Format Bazel code"
+    info "${TAB}-c|--cpp         Format cpp code"
+    info "${TAB}-a|--all         Format all (C++/Python/Bazel)"
+    info "${TAB}-h|--help        # Show this message"
```

Comment:
There are two indent styles in this pull request, 2-spaces and 4-spaces. Please stick to one style. And we can unify the whole repo later.

Reply:
Ok